### PR TITLE
Updated to reflect trunk instead of devel

### PR DESCRIPTION
**Pull request by nickylimjj**

### CODE REVIEW

File: docs/en-US/references/default-credentials.md [hunk 1]
```diff
@@ -34,10 +34,8 @@ __Password:__ `vagrant`
...
-#### WordPress Develop
-* LOCAL PATH: vagrant-local/www/wordpress-develop
-* VM PATH: /srv/www/wordpress-develop
-* /src URL: `http://src.wordpress-develop.test`
-* /build URL: `http://build.wordpress-develop.test`
-* DB Name: `wordpress_develop`
+#### WordPress Trunk  
+* LOCAL PATH: vagrant-local/www/wordpress-trunk
+* VM PATH: /srv/www/wordpress-trunk
+* DB Name: `wordpresstrunk`
```
Comment (tomjn) [inline docs/en-US/references/default-credentials.md]: Should this be `wordpress_trunk`?

**Comment by tomjn** [comment]
Thanks for the changes @nickylimjj, I think the db name needs a double check, once that's done we can merge

**Comment by nickylimjj** [comment]
I checked when I start vvv.test. I can take a screenshot and confirm

**Comment by nickylimjj** [comment]
> Thanks for the changes @nickylimjj, I think the db name needs a double check, once that's done we can merge
@tomjn 
wp_db

**Review by tomjn — APPROVED** [review]

**Pull request merged**